index.html

```diff
@@ -234,52 +261,68 @@ <h1 class="about_myself_heading">About me</h1>
       </ul>
     </section>
 
-    <section class="contact_section">
-      <h3>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h3>
-
-      <form
-        action="https://formspree.io/f/xgedowvw"
-        method="POST"
-        class="contact_form"
-      >
-        <input
-          class="name"
-          type="text"
-          placeholder="Full name"
-          required
-          maxlength="30"
-        />
-        <hr class="horizontal_line" />
-        <input
-          class="email"
-          type="email"
-          placeholder="Email address"
-          required
-        />
-        <hr class="horizontal_line" />
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          cols="30"
-          rows="10"
-          maxlength="500"
-          placeholder="Enter text here"
-          required
-        ></textarea>
-
-        <button class="submit_button" type="submit">Get in touch</button>
-      </form>
-    </section>
-
-    <hr class="horizontal_line horizontal_line_3" />
-
     <footer class="footer">
-      <div class="social-media social-media-footer">
-        <ul class="social-media-links">
+      <section class="contact_section" id="contact">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+
+        <form
+          action="https://formspree.io/f/xgedowvw"
+          method="POST"
+          class="contact_form"
+        >
+          <input
+            class="full_name"
+            type="text"
+            placeholder="Full Name"
+            name="full_name"
+            required
+            maxlength="30"
+          />
+
+          <input
+            class="first_name"
+            type="text"
+            placeholder="First Name"
+            name="first_name"
+            maxlength="30"
+            required
+          />
+
+          <input
+            class="last_name"
+            type="text"
+            placeholder="Last Name"
+            name="last_name"
+            maxlength="30"
+            required
+          />
```

Comment:
- [ ] Kindly please be mindful of the project requirements.  Let's make these form fields just one that contains the whole name of the person.

![image](https://user-images.githubusercontent.com/57421823/149312440-0b9f5ae8-4fae-49bf-9558-9e235266c440.png)

Reply:
I was also confused about this part of the design, the input fields are different in the mobile and desktop versions. In the mobile version, there is the only full name, email, and text area. But in the desktop version of the UI, there is First name, Last name, Email, and text area. I was just trying to make it the template.